### DISCUSSION TITLE
Access Nginx UI with domain

### DISCUSSION THREAD

**Discussion by Marc-Proux** [NginxProxyManager/nginx-proxy-manager#3491] [Q&A]
Hi,

I'm new to using Nginx Proxy Manager on my TrueNAS computer. I would like to be able to access my Nginx UI from other networks so that I can work on it at any time. I have set it up, but when I try to access the page, I encounter a 504 Bad Gateway error.

Here is how I configured my Nginx:
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/119617467/beb7f90c-0e44-4e44-be8d-3aae3552ca89)

Does anyone have a solution?

Thanks a lot.

**Comment by lastsamurai26** [comment]
You want to open the NPM UI with a domain instead of ip:81 ? 
or do you want to open your TrueNAS with a domain? 

My question is where is the NPM Docker installed, the port 30020 seems strange to me. Is the NPM running on the same machine ?
then you only need to release port 443 and 80 in NPM and set the website 127.0.0.1 -> 81 to http then this should work.

Otherwise show me your docker compose from NPM

**Reply by Marc-Proux** [reply]
When I installed Nginx with TrueNAS, I no longer had the option to set ports 30020 for the admin panel, 30021 for HTTP, and 30022 for HTTPS. Therefore, I configured my NAT to redirect port 80 to 30021 and port 443 to 30022. The idea is to open NPM UI with a domain instead of ip:30020.

**Reply by lastsamurai26** [reply]
ok anyway 
try http 127.0.0.1 81 

otherwise I can't help, I'm only a user myself. 
I have NPM as backend (3102, 3103, 3104) and an Apache on the front, which forwards the domain to the backend 

Normally you do not need to open port 81, as this should be done by the web server 

This means that if you call up https://npm.domain.de:30022, the NPM should recognise this and redirect you to 127.0.0.1:81 as you have specified the domain npm.domain.de for the NPM.
